chainbase/src/main/java/org/tron/common/utils/DBConfig.java

```diff
@@ -124,4 +124,8 @@
   @Getter
   @Setter
   private static String transactionHistoreSwitch;
+
+  @Getter
+  @Setter
+  private static boolean saveInternalTx;
```

Comment:
don't need to pass the value to dbconfig. You could directly use config from parameter

Reply:
@wubinTron Thanks for your suggestions. I have update the PR. Please check again.